src/lib/serviceRoutinesV2/logLevelTreat.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+*
+* Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Ken Zangelin
+*/
+#include <string>
+#include <vector>
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+
+#include "common/statistics.h"
+#include "common/clockFunctions.h"
+
+#include "ngsi/ParseData.h"
+#include "rest/ConnectionInfo.h"
+#include "rest/orionReply.h"
+#include "rest/OrionError.h"
+#include "rest/uriParamNames.h"
+#include "serviceRoutinesV2/logLevelTreat.h"
+#include "alarmMgr/alarmMgr.h"
+
+
+
+/* ****************************************************************************
+*
+* logLevelTreat -
+*/
+std::string logLevelTreat
+(
+  ConnectionInfo*            ciP,
+  int                        components,
+  std::vector<std::string>&  compV,
+  ParseData*                 parseDataP
+)
+{
+  std::string  level   = ciP->uriParam[URI_PARAM_LEVEL];
+  const char*  levelP  = level.c_str();
+
+  if (level == "")
+  {
+    ciP->httpStatusCode = SccBadRequest;
+    alarmMgr.badInput(clientIp, "no log level in URI param");
```

Comment:
The release is for clientIp.
Simply a correct request from that IP and the badInput should be released